report_viewer/apps/viewer/middleware/report_views_middleware.py

```diff
@@ -46,11 +46,7 @@ def user_fingerprint(self, request: HttpRequest, secret_key: str) -> str:
         Returns:
             str: An amalgamated string of the user agent, client ip, and secret key
```

Comment:
"secret key" mentioned here has been removed below.

The "secret key" is also mentioned in other docstrings in this model.

These should all be remove if the `secret_key` is no longer used.